src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Renderer.js

```diff
@@ -67,7 +67,7 @@ class Renderer extends React.Component<Props> {
                 dataPath={itemDataPath}
                 error={error}
                 formInspector={formInspector}
-                key={schemaKey}
+                key={schemaKey + '_' + schemaField.type}
```

Comment:
I moved the key into the `Renderer` component because we had a key here already. I successfully tested the changes with the template provided in https://github.com/sulu/sulu/issues/6613#issue-1238789465